src/main/webapp/config.js

```diff
@@ -88,6 +90,18 @@ const hideSiteInputs = () => {
     handleFormChange();
 };
 
+const toggleAdvancedOptions = () => {
```

Comment:
show hide the options, and rotate the chevron
we use position instead of anything normal like not showing or injecting the options because Jenkins maintains a JSON tree structure behind the scenes and this was the most robust way to do it

Reply:
Love the comments explaining the improvements you've done :)